picli/config.py

```diff
@@ -23,8 +23,8 @@ def _find_base_dir(self, config):
         configuration file. This will be the base directory
         that all other methods will assume.
         """
-        base_dir = os.path.dirname(os.path.dirname(config))
-        if not base_dir:
+        base_dir = os.path.normpath(os.path.join(os.path.abspath(config), '../..'))

```

Comment:
you might want to also consider adding path.realpath which removes symlinks and gives you an absolute path... resolving the .././ stuff...

Also path.sep here would make this work on both windows and *nix paths

Should also resolve path.expandvars and path.expanduser to support paths like `$HOME/somedir/etc/` and `~/somedir/things`

I usually put the path handling things into util also, I'm not sure why python doesn't have a path.make_it_so() method that does all that for you... but it doesn't

Reply:
Yea i think that would be a good idea. I'll probably save that for a separate PR though